en/includes/guides/identity-verification/manage-identity-verification-provider.md

```diff
@@ -0,0 +1,60 @@
+## Manage an Identity Verification Provider
+
+This section provides instructions on how to manage Identity Verification Providers (IDVPs) once they are created.
+
+### View an Identity Verification Provider
+
+To view the list of IDVPs added to the {{ product_name }}:
+
+1. On the {{ product_name }} Console, go to **Connections**.
+
+2. Select **Identity-Verification** filter tag to filter out the created IDVPs.
+   ![idvp-list]({{base_path}}/assets/img/guides/identity-verification/view-idvp.png)
+
+### Edit an Identity Verification Provider
+
+An administrator can update an IDVP via the {{ product_name }} Console.
+
+To edit an IDVP:
+
+1. On the {{ product_name }} Console, go to **Connections**.
+
+2. Find the IDVP and click **Setup up**.
+   ![idvp-list]({{base_path}}/assets/img/guides/identity-verification/view-idvp.png)
+
+3. Update the information on the IDVP.
+
+4. Click **Update** to save.
+
+### Disable an Identity Verification Provider
+
+An administrator can disable/enable an IDVP via the {{ product_name }} Console.
+
+To disable an IDVP:
+
+1. On the {{ product_name }} Console, go to **Connections**.
+
+2. Find the IDVP and click **Setup up**.
```

Comment:
```suggestion
2. Find the IDVP and click **Setup**.
```

Reply:
Refer to: https://github.com/wso2/docs-is/pull/4803/files#r1792857699

Reply:
https://github.com/wso2/docs-is/pull/4803#discussion_r1792863663